components/recommendation/RecommendationArticle.vue

```diff
@@ -18,15 +18,14 @@ function checkIntersection([{ isIntersecting }]: [{ isIntersecting: boolean }]):
 }
 
 const clipboard = useClipboard()
-async function copyLink(url: string, event: Event): void {
-  event.preventDefault()
+async function copyLink(url: string): void {
```

Comment:
Since this is no longer within the `a` tag we don't need to prevent the default action anymore.